.travis.yml

```diff
@@ -1,6 +1,16 @@
 language: node_js
 node_js:
-  - "0.10"
-  - "0.11"
-  - "0.12"
-
+  - "9.11.1"
+dist: trusty
+cache: npm
+before_install:
+  - npm i -g npm@6.0.1
```

Comment:
any reason to pin this specific version of npm?

Reply:
Not in particular, I've upgraded this to `6.4.1` 👍